app/controllers/lottery_controller.rb

```diff
@@ -44,20 +35,45 @@ def run_intelligent_assignment
 
   private
 
+  def construct_priority_info(sign_up_topics, teams)
+    priority_info = []
+    # Exclude any teams already signed up
+    teams_not_signing_up = teams.select{|team| SignedUpTeam.where(team_id: team.id, is_waitlisted: 0).any?}
+    teams_not_signing_up.each do |team|
+      # Grab student id and list of bids
+      bids = [0]
```

Comment:
I believe starting the bids with 0 will affect the matrix in the merge_bids_from_different_previous_teams method.

Reply:
Could you illustrate more on this part? Because what I thought was that if bids array contains only [0] then priority_info will not be updated?

Reply:
If you add a zero to the bids array, and then a bid_record is found and the bid_record.priority is added to the bids array, the indices of the priorities will be off by 1 so that when merge_bids_from_different_previous_teams is called, it might see that zero in the 0th index of the user_bidding_info and think that zero corresponds to the priority for the sign_up_topic at the 0th index.